group_vars/mongodb/public.yml

```diff
@@ -3,6 +3,11 @@ mongodb_daemon_name: "mongodb"
 mongodb_net_bindip: "0.0.0.0"
 mongodb_security_authorization: "enabled"
 mongodb_storage_dbpath: "/var/lib/mongodb"
+# We do not enable the quota, but noticed that it gets enforced at least in some
+# contexts anyway.  Setting this to 36 results in a quota of 64 GB per database,
+# which ought to be enough for anybody.  (Currently, our by far biggest instance
+# 14 GB for the module store.)
```

Comment:
@smarnach Missing words: "... our by far biggest instance **is using** 14 GB..."